### DISCUSSION TITLE
Daily 13/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#188] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer, luego de subir la conexion de registración hice la parte responsive de la activity, ya está subido al repositorio. Por la tarde participé de la clase sincrónica del ISPC.
Hoy voy a estar haciendo la parte de validaciones de los campos de registración y estudiando para el cuestionario que se subió en la plataforma.

**Comment by BraianTroncoso** [comment]
Ayer estuve estudiando mis tareas asignadas para realizarlas, estuve con mucho trabajo y carga horaria de clases extras. Hoy me desocupe tambien tarde, pude realizar el examen y aprobarlo que nos dejaron hasta el lunes. También avancé con otras dudas que tenia referidas al proyecto.
